compose/foundation/foundation/src/nativeMain/kotlin/androidx/compose/foundation/Overscroll.native.kt

```diff
@@ -27,46 +27,28 @@ import androidx.compose.ui.unit.Velocity
 @Composable
 internal actual fun rememberOverscrollEffect(): OverscrollEffect {
     return remember {
-        DesktopEdgeEffectOverscrollEffect()
+        NativeOverscrollEffect() // TODO: Split between UIKit and MacOS
     }
 }
 
 @OptIn(ExperimentalFoundationApi::class)
-private class DesktopEdgeEffectOverscrollEffect() : OverscrollEffect {
-    // TODO: [1.4 Update] implement it properly after API changes and remove commented old code
-
+private class NativeOverscrollEffect() : OverscrollEffect {
     override fun applyToScroll(
         delta: Offset,
         source: NestedScrollSource,
         performScroll: (Offset) -> Offset
     ): Offset {
+        performScroll(delta)
         return Offset.Zero
```

Comment:
From the doc of this function:

```
This function should return the
     * sum of all the delta that was consumed during this operation - both by the overscroll and
     * [performScroll].
```

So maybe returning Offset.Zero is not correct? We can at least return the result of `performScroll`?
It says  to add `overscroll` deltas too, but I'm not sure where we can get it here.

Reply:
Thanks, I will check

Reply:
Yes we just need to return performScroll(delta) in our case. Android has many code in its implementation, I believe it can be applied for our targets as well but it requires thorough testing, so we need to add TODO: at least for it

Reply:
Done